config/config.go

```diff
@@ -14,12 +14,18 @@ const (
 
 type Config struct {
 	Listen          *string                   `yaml:"listen" validate:"required"`
+	TLSConfig       *TLSConfig                `yaml:"tls"`
 	ShutDownTimeout *time.Duration            `yaml:"shutdown_timeout"`
 	ExporterConfigs map[string]ExporterConfig `yaml:"exporters" validate:"required,dive"`
 	AccessLogConfig *AccessLogConfig          `yaml:"access_log"`
 	ErrorLogConfig  *ErrorLogConfig           `yaml:"error_log"`
 }
 
+type TLSConfig struct {
+	CertFile *string `yaml:"certfile"`
```

Comment:
I think `CertFile` and `KeyFile` should be required if TLSConfig specified.
Could you add `validate: "required"` keyword to these?